addons/udes_stock/models/stock_inventory.py

```diff
@@ -38,6 +45,14 @@ class StockInventory(models.Model):
         "Start Date Scanning",
         help="The date operator started scanning."
     )
+    user_id = fields.Many2one(
+        "res.users",
+        string="Checked Owner",
+        readonly=True,
+        states={"draft": [("readonly", False)], "waiting": [("readonly", False)]},
+        help="Specify Owner to focus your stock check on a particular user.",
```

Comment:
Nit: we can make the help string simpler e.g "Stock check owner"

Reply:
As i said previously its a copy paste from udes_stock_count